src/gameHandlerClass.ts

```diff
@@ -35,4 +35,11 @@ export class GameHandler {
       ? this._playerTwo
       : this._playerOne;
   }
-}
+
+  useMango(): void | Error{
```

Comment:
ideally there should be one more method: "can I use mango" or something like that, that would be used to render if "use mango" button will be disabled or enabled :)